src/index.js

```diff
@@ -5,14 +5,16 @@ import { StyleSheetServer } from 'aphrodite/no-important';
 import ampPage from './ampPage';
 
 // eslint-disable-next-line import/prefer-default-export
-export const renderReactAmpWithAphrodite = (name, component, ampOptions) => hypernova({
+export const renderReactAmpWithAphrodite = (name, component, ampOptions = {}) => hypernova({
   server() {
     return (props) => {
       const { html, css } = StyleSheetServer.renderStatic(() => (
         ReactDOMServer.renderToString(React.createElement(component, props))
       ));
 
-      const style = `<style amp-custom>\n${css.content}\n</style>`;
+      const style =
+        `<style amp-custom>\n${ampOptions.prependCss || ''}${css.content}${ampOptions.appendCss || ''}\n</style>`;
```

Comment:
please don't arbitrarily add line breaks; max-len ignores long lines with strings, on purpose.

would it be a bit cleaner to do `const appendCSS = ampOptions.prependCSS || ''`? (also, can we properly capitalize the initialism? we don't need to appease a java-style camelCasing algorithm)